src/main/java/com/cachinginapp/enterprise/CachingInController.java

```diff
@@ -63,7 +66,14 @@ public String saveCache(Cache cache){
     @GetMapping("/cache")
     @ResponseBody
     public List<Cache> fetchAllCaches(){
-        return cacheService.fetchAll();
+        try{
+            return cacheService.fetchAll();
+        } catch (Exception e) {
+            logger.log(Level.WARNING, "No caches were fetched!");
+            throw new RuntimeException(e);
+        }
+
+
```

Comment:
added logger and try catch in case of exception

Reply:
I wonder if this is truly exception worthy... or if simply returning an empty List would be better.  That feels like a more common approach.
Either way, logging is a good idea.